index.bs

```diff
@@ -224,13 +224,15 @@ following steps:
  <li>Run [=retrieve the client hints set=] with |settingsObject|.
  <li>For each [=client hints token=] |lowEntropyHint| in the registry's [=low entropy hint table=], [=set/append=] |lowEntropyHint| to |hintSet|.
  <li>If |request|'s [=request/client=] is not null, then for each [=client hints token=] |requestHint| in
-|request|'s [=environment settings object/client hints set=], [=set/append=] |requestHint| to
+|settingsObject|'s [=environment settings object/client hints set=], [=set/append=] |requestHint| to
 |hintSet|.
  <li>For each |hintName| in |hintSet|:
  <ol>
    <li>If |request| is not a [=navigation request=] for a "document" [=request/destination=] and if the result of running [[permissions-policy#algo-should-request-be-allowed-to-use-feature]]
    given |request| and |hintName|'s associated feature in [[#policy-controlled-features]] returns `false`, then continue to next |hintName|.
+   <li>If |hintName| should be ommitted (see [=Security and Privacy considerations=]) then continue.
    <li>Let |value| be the result of running [=find client hint value=] with |hintName|.
+   <li>If |hintName| should be lied about (see [=Security and Privacy considerations=]) then let |value| equal that lie.
```

Comment:
Maybe "Potentially modify |value| in a [implementation defined](https://infra.spec.whatwg.org/#implementation-defined) way"?